tfhe/src/shortint/server_key/mod.rs

```diff
@@ -54,14 +57,77 @@ impl Display for CheckError {
 
 impl std::error::Error for CheckError {}
 
+#[derive(Clone, Debug, PartialEq, Serialize, Deserialize)]
+pub enum ShortintBootstrappingKeyBase {
+    Classic(FourierLweBootstrapKeyOwned),
+    // TODO: do not serialize ThreadCount, manage that case separately for ser/de and repopulate
```

Comment:
Is this todo meant for now or later ?

Reply:
If I had time I would have done it right away, in practice I don't have the time before vacations to do that properly and prime Q so I have follow-up issues see https://github.com/zama-ai/tfhe-rs-internal/issues/126 https://github.com/zama-ai/tfhe-rs-internal/issues/125

Reply:
ok